management/management_test.go

```diff
@@ -21,21 +21,28 @@ var (
 	debug                 = os.Getenv("AUTH0_DEBUG")
 	httpRecordings        = os.Getenv("AUTH0_HTTP_RECORDINGS")
 	httpRecordingsEnabled = false
-	m                     = &Management{}
+	api                   = &Management{}
 )
 
-func init() {
-	httpRecordingsEnabled = httpRecordings == "true" || httpRecordings == "1" || httpRecordings == "on"
-	initTestManagement()
+func envVarEnabled(envVar string) bool {
+	return envVar == "true" || envVar == "1" || envVar == "on"
 }
 
-func initTestManagement() {
+func TestMain(m *testing.M) {
```

Comment:
Replacing the `init` func with `TestMain` to follow golang testing best practices.